ops/services/app_service/main.tf

```diff
@@ -10,48 +10,87 @@ locals {
       "DOCKER_REGISTRY_SERVER_USERNAME" = data.terraform_remote_state.global.outputs.acr_simeplereport_name
       "WEBSITES_PORT"                   = "8080"
       "WEBSITE_DNS_SERVER"              = "168.63.129.16" # https://docs.microsoft.com/en-us/azure/app-service/web-sites-integrate-with-vnet#azure-dns-private-zones
-      "WEBSITE_VNET_ROUTE_ALL"          = "1"
   })
 }
 
-resource "azurerm_app_service_plan" "service_plan" {
+# migrate this to azurerm_service_plan.service_plan to enable migration of api service
```

Comment:
Was this comment block intentional? I'm fine leaving it if it is for a documentary purpose.